src/components/SideBar.tsx

```diff
@@ -30,8 +31,10 @@ const SideBar = (props: Types.SideBarProps): JSX.Element => {
         </p>
       </div>
       <div id="waypoints">
-        <h1>cBioPortal Instances</h1>
-        <FilterOptions stateManager={props.stateManager} />
+     <div id = "tableHeader"> 
+      <h1> cBioPortal Instances: <Renderers.tableRowCounterTwo /> </h1>
+    </div>        
+<FilterOptions stateManager={props.stateManager} />
```

Comment:
We should try to align this in a better format.
```suggestion
      <div id = "tableHeader"> 
        <h1> cBioPortal Instances: <Renderers.tableRowCounterTwo /> </h1>
      </div>        
      <FilterOptions stateManager={props.stateManager} />
```